autogpt_platform/backend/backend/blocks/__init__.py

```diff
@@ -24,7 +25,7 @@
     AVAILABLE_MODULES.append(module)
 
 # Load all Block instances from the available modules
-AVAILABLE_BLOCKS = {}
+AVAILABLE_BLOCKS: dict[str, Type[Block]] = {}
 
 
 def all_subclasses(clz):
```

Comment:
I know this isn't changed but python doesn't like `cls` here or something? It looks fine for me in vs code when changed.

Reply:
I've been using cls in the below code, this is to avoid overriding variables.
I've renamed them and the variables below it to something else.